### PR TITLE
85 refactoring screen methods

### LINKED ISSUE #85: Refactoring of Screen.start() methods
These methods are simply too long

### PR DESCRIPTION
**Pull request by mdingjan**
Refactored the screen methods of SplashScreen, SettingsScreen and StartScreen.
Please have a look :)
Will close #85

<organisation, layout

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/group9/gui/SettingsScreen.java [hunk 1]
```diff
@@ -95,4 +65,61 @@ public void handle(final ActionEvent event) {
...
+    }
+
+    /**
+     * This method adds text to buttons and give them a location on the grid.
+     * @param name Name of the button.
+     * @param colum Colum index on the grid.
+     * @param row Row index on the grid.
+     * @return the created button.
+     */
+    private static Button createButton(String name, int colum, int row) {
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/SettingsScreen.java]: Please rename `colum

File: src/main/java/nl/tudelft/ti2206/group9/gui/StartScreen.java [hunk 2]
```diff
@@ -11,84 +11,103 @@
...
- * @author Maikel and Robin
+ * @author Maikel, Mitchell and Robin
  *
  * A startscreen with different options/buttons like a options menu, startbutton
  * and exit button.
  */
 @SuppressWarnings("restriction")
 public final class StartScreen {
-
+
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/StartScreen.java]: Well this is a silly addition. :P
<layout

File: src/main/java/nl/tudelft/ti2206/group9/gui/StartScreen.java [hunk 3]
```diff
@@ -11,84 +11,103 @@
...
+    }
+    
+    /**
+     * This method adds text to buttons and give them a location on the grid.
+     * @param name Name of the button.
+     * @param colum Colum index on the grid.
+     * @param row Row index on the grid.
+     * @return the created button.
+     */
+    private static Button createButton(String name, int colum, int row) {
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/StartScreen.java]: Parameters could be final.
<interface

**Comment by mpsijm** [comment]
The refactoring looks good, I will continue in #95 :)
Once the checkstyle warnings are resolved, merge away!
<review,layout,documentation

**Comment by MathiasMeuleman** [comment]
Looks good! Tnx for fixing the checkstyle warnings.
Will merge

<review

**Pull request merged**